Overview.bs

```diff
@@ -962,10 +963,13 @@ should interpret and process the fields of the object as follows:
     resend the request that triggered this response but use the new codepoint ordering provided in
     this response.
 
-5. If <code>original_axis_space</code> is set then update the saved original axis space with the value
+5. If <code>original_features</code> is set and the client has opted to save them then update the saved
```

Comment:
This list outlines the fields a client should process. In the edited step 5 description: 

> If <code>original_features</code> is set and the client has opted to save them then update the saved original feature list with the value specified in this field.

the last part of the sentence "then update the saved original feature list" is a bit ambiguous and may need to be clarified.

The rest of the changes LGTM.